meddlr/data/transforms/transform.py

```diff
@@ -364,3 +365,211 @@ def __call__(self, kspace, maps, target, fname, slice_id, is_fixed, acceleration
         target = target.squeeze(0)
 
         return masked_kspace, maps, target, mean, std, norm
+
+
+class MotionDataTransform:
+    """
+    Data Transformer for training unrolled reconstruction models.
+
+    This is for emulating 2D roto-translational motion corrupted MR scans. 
+    """
+
+    def __init__(
+        self,
+        cfg,
+        mask_func,
+        is_test: bool = False,
+        add_noise: bool = False,
+        add_motion: bool = False,
+        angle: float = 0,
+        translation: float = 0,
+        nshots: int = 0,
```

Comment:
is a default value of `0` ever valid? if not, set it to `None` and raise a ValueError if it is not set

Reply:
@ad12 Should this ValueError be raised if a parameter is set to None? Sorry, just trying to better understand. I added an Optional to the angle and translation parameters for the motion corruption function, so the function can handle None parameters for angle and translation right? So, would I still raise that ValueError? Or do I raise it for something else?

Reply:
Would nshots be checked to see if it was set? And thus would that raise a ValueError? Also, would the type annotation for nshots in this function be just nshots: int? Thanks, Deepro.